### PR TITLE
Web scraper: does not use sessionPool in dev mode

### PR DESCRIPTION
**Pull request by jbartadev**
Options `useSessionPool` and `persistCookiesPerSession` are not available in `DEVELOPMENT` mode for the `web scraper`.

### CODE REVIEW

File: web-scraper/src/crawler_setup.js [hunk 1]
```diff
@@ -267,10 +267,13 @@ class CrawlerSetup {
...
-            const cookiesToSet = tools.getMissingCookiesFromSession(session, this.input.initialCookies, request.url);
+            const cookiesToSet = session
+                ? tools.getMissingCookiesFromSession(session, this.input.initialCookies, request.url)
+                : this.input.initialCookies;
             if (cookiesToSet && cookiesToSet.length) {
                 // setting initial cookies that are not already in the session and page
-                session.setPuppeteerCookies(cookiesToSet, request.url);
+                if (session) session.setPuppeteerCookies(cookiesToSet, request.url);
+                // TODO: We have to change this when there is an option to define blocked status codes in sessionPool
                 await page.setCookie(...cookiesToSet);
```
Comment (mnmkng) [inline web-scraper/src/crawler_setup.js:277]: This makes it look like the TODO belongs with the `page.setCookie`, which is confusing. It's also not very clear from the comment what needs to be done.

**Comment by mnmkng** [comment]
Please also update the Run mode tooltip and the log message that's printed in development mode.

File: web-scraper/src/crawler_setup.js (hunk 1 above)
Comment (jbartadev) [reply web-scraper/src/crawler_setup.js]: Changed.

**Comment by jbartadev** [comment]
> Please also update the Run mode tooltip and the log message that's printed in development mode.

Updated.

**Review by mnmkng — APPROVED** [review]

**Pull request merged**